importers/src/bible/NEUE/ref-parser-test.ts

```diff
@@ -1,4 +1,13 @@
-import { BibleReferenceParser, getReferencesFromText } from '@bible-engine/core';
```

Comment:
maybe add a comment at the top of the file, that this is just a script for manual testing of regex?